test/__snapshots__/StatsTestCases.test.js.snap

```diff
@@ -3200,6 +3200,14 @@ Child enforce-min-size:
         > ./ main"
 `;
 
+exports[`StatsTestCases should print correct stats for split-chunks-issue-7230 1`] = `
+"         Asset      Size  Chunks             Chunk Names
+core.bundle.js  6.59 KiB       0  [emitted]  core
+main.bundle.js  6.83 KiB       1  [emitted]  main
+Entrypoint core = core.bundle.js
+Entrypoint main = core.bundle.js main.bundle.js"
```

Comment:
This is a non-working bundle.

both core and main include the runtime. Using both will throw an error about missing modules at runtime. main can't use the modules in core.

Reply:
I thought the original pull request needed just a rebase. I'll look into it. Do you have any suggestions what needs to be done?

Also - I'm not sure why there was such a complex test function https://github.com/webpack/webpack/pull/8614/files#diff-741d374868f6de786477bdcfbaf6fa81R32, the test can be simpler, right?

And looking at it some more, maybe `Preserve existing chunk as an entrypoint when it has matching modules` is not a very clear description, or is it? I'd suggest something like `Preserve entrypoint when it has same name name as chunk`?

Reply:
@sokra Fixed. I fixed and rewrote the test to be simpler and more readable & improved the description.